src/expandableCalendar/calendarProvider.js

```diff
@@ -31,6 +32,8 @@ class CalendarProvider extends Component {
     onMonthChange: PropTypes.func,
     /** whether to show the today button */
     showTodayButton: PropTypes.bool,
+    /** The button top position */
+    buttonTopPosition: PropTypes.number,
```

Comment:
shouldn't it be bottomPosition? 
also I would name the prop `todayButton...Position`

Reply:
It the button's Y position. You think that's bottom?